sdk/core/AzureCore/Source/Pipeline/PipelineClient.swift

```diff
@@ -59,6 +59,7 @@ open class PipelineClient {
     public var baseUrl: URL
     public var logger: ClientLogger
```

Comment:
Unrelated, but I just noticed we accept a logger here, but the 1commonOptions` also contains a logger. Maybe we can clean this up.